images/talos/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.10-bullseye
+
+ARG VERSION=${VERSION:-4.0.0}
+
+RUN apt update && apt install -y \
+        apt-transport-https \
+        bzip2 \
+        ca-certificates \
+        git \
+        gnupg \
+        openjdk-11-jdk-headless \
+        wget \
+        zip && \
+    rm -r /var/lib/apt/lists/* && \
+    rm -r /var/cache/apt/* && \
+    # Google Cloud SDK: use the script-based installation, as the Debian package is outdated.
+    curl https://sdk.cloud.google.com > install.sh && \
+    bash install.sh --disable-prompts --install-dir=/opt && \
+    rm install.sh
+
+ENV PATH=$PATH:/opt/google-cloud-sdk/bin
+
+# pull the talos repo and build \
+RUN git clone https://github.com/populationgenomics/automated-interpretation-pipeline.git
+
+WORKDIR automated-interpretation-pipeline
+RUN python3 -m pip install .[cpg]
```

Comment:
Just pip install straight from the github repo, and add a ` --no-cache-dir` arg.

Reply:
I wasn't comfortable doing this before, as I'm using the path to specific scripts for the various steps of the process. I've added an entrypoint to each part of the process in a separate PR so this is doable

Reply:
Fair fair, otherwise you'd want to checkout with `--depth 1` to avoid git downloading and storing the history